libexec/bats-core/bats-exec-test

```diff
@@ -40,24 +40,21 @@ fi
 BATS_TEST_DIRNAME="${BATS_TEST_FILENAME%/*}"
 BATS_TEST_NAMES=()
 
+# Shorthand for source-ing files relative to the BATS_TEST_DIRNAME,
+# optionally with a .bash suffix appended. If the argument doesn't
+# resolve relative to BATS_TEST_DIRNAME it is sourced as-is.
 load() {
-  local name="$1"
```

Comment:
Apologies for my naivety  — I usually use `:-`, can you explain the behaviour here for me please?

Reply:
`:?` fails the script if the variable is "null or not set", so this just ensures the user has in fact passed an argument to `load`.

Reply:
What's the significance of the `name` value in here, a default if `$1` is an empty string?

Reply:
The text after `:?` is the error that gets reported, and looking at it again I think the default error message is preferable so I've removed it. You can experiment with this syntax, try `echo ${foo:?}`, see also [Shell Parameter Expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html).